### DISCUSSION TITLE
Como configurar una rutina desde el creador de task (botón +)

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#517] [2. Questions and Answers] [answered]
Hola a todos

Quisiera saber si existe una forma en configurar una rutina de una tasks que se crea desde el botón +, vista tasks.

Lo que necesito hacer es que cada vez que se crea una tasks, desde la vista tareas, debe cambiar la imagen del canal

Gracias =>

**Comment by jdurandpaz** [comment]
@NicolasPLG pero no te aparece el formulario requerido cuando creas la task de esa vista?

**Reply by NicolasPLG** [reply]
@jdurandpaz no es para un proyecto interno de Cotalker, es para un cliente.

Los pasos son

1.- crear la task desde el +
2.- agregar una nota desde la tarea creada

Eso

**Comment by guillevalin** [comment, marked as answer]
@NicolasPLG 

Si configuras el start form de una SM, aquí:
![image](https://user-images.githubusercontent.com/34345309/139132516-290ce64a-00df-4e6a-957d-c3a8d1aaa028.png)

Podrás usar el FABbutton en la lista de canales:
![image](https://user-images.githubusercontent.com/34345309/139132574-560407e0-3603-44ee-bc90-4111ff442914.png)

Pero también el botón + en la lista de tareas:
![image](https://user-images.githubusercontent.com/34345309/139132615-f59bae14-fc0f-40ac-829a-117773999dac.png)

Mayor información en:
https://doc.cotalker.com/docs/documentation/admin/workflows/admin_workflow_required_survey#required-survey-for-a-new-task
